src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -465,6 +466,7 @@ public void canCreateAnItemWithManyProperties()
     itemToCreate.put("permanentLoanTypeId", canCirculateLoanTypeID);
     itemToCreate.put("temporaryLocationId", annexLibraryLocationId.toString());
     itemToCreate.put("tags", new JsonObject().put("tagList", new JsonArray().add(TAG_VALUE)));
+    itemToCreate.put("itemDamagedStatusDescription", damagedDescription);
```

Comment:
Should we allow a damage description without a damage status being set? How are these properties intended to work together, if at all?  Maybe "damage" should be a sub-object with certain required fields? What do you think @marcjohnson-kint?

Reply:
Hi @mreno-EBSCO, good point. I agree that those three properties `(itemDamagedStatusId`, `itemDamagedStatusDate`, `itemDamagedDescription`) should be handled only together.

What about following structure?
```javascript
"damage": {
  "type": "object",
  "description": "Item damage information",
  "properties": {
    "statusId": {
      "description": "Damage status ID",
      "type": "string"
    },
    "statusDate": {
      "description": "Damage date",
      "type": "string"
    },
    "description": {
      "type": "string",
      "description": "Holds information about how an item is damaged"
    }
  },
  "additionalProperties": false,
  "required": [
    "statusId",
    "statusDate",
    "description"
  ]
}
```

But then, it will be breaking changes for UI. We should let UI guys know about it. Not aware about process in this case. 

Let's wait response from @marcjohnson-kint.

Reply:
@mreno-EBSCO @bohdan-suprun I don't think I have good answers for you, as these questions feel like requirements gaps.



> Should we allow a damage description without a damage status being set? How are these properties intended to work together, if at all?

I think these are good questions for Charlotte as a comment on the issue.

> Maybe "damage" should be a sub-object with certain required fields? What do you think @marcjohnson-kint?

> What about following structure?

As above, I would prefer these properties were grouped together, however they are not and changing that would be compatibility breaking, and I don't really want to do that right now, unless we feel it is necessary?

Also as above, my interpretation of Scenario 3 is that the date is for any changes to damage information, not just the status, so should really be named more generally.

We could describe this challenge on the issue and let Cate and Charlotte decide if we want to feel the pain now or later (where it will be more expensive because we would need to migrate data due to Chalmers being in production).

@skoczko What do you think?

> But then, it will be breaking changes for UI. We should let UI guys know about it. Not aware about process in this case.

The process involves coordinating changes to all dependent modules in the reference system, which for items is probably quite a few, especially as this would likely also mean a breaking change to all dependents of the `inventory` interface too.